x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -398,6 +398,7 @@ export interface GetGlobalExecutionKPIParams {
   dateStart: string;
   dateEnd?: string;
   filter?: string;
+  namespaces?: Array<string | undefined>;
```

Comment:
Is it possible to make this type a bit stricter? (I.e. just  `Array<string>`). I see that `getNameSpace` could return undefined, could we filter those entries out or do they have some significance being undefined?

Reply:
The Default namespace is referred to as `undefined` internally. As far as I can tell, this changed recently; it used to just be the string `default`.